sortWords.py

```diff
@@ -0,0 +1,18 @@
+# Program to sort alphabetically the words form a string provided by the user
```

Comment:
@Udula-eranda 
The PR is out of scope!
Please read the repository read me file for details!